src/theme/foundations/textStyles.ts

```diff
@@ -15,43 +15,28 @@ export const textStyles = {
     letterSpacing: { base: '0.03rem', md: '0.04rem' },
     color: 'body'
   },
-  header1: {
-    fontFamily: 'heading',
-    fontWeight: 700,
-    fontSize: { base: '1.875rem', md: '2.125rem' },
-    letterSpacing: { base: '0.03rem', md: '0.04rem' },
-    lineHeight: 'normal',
-    color: 'body'
-  },
-  header2: {
-    fontFamily: 'heading',
-    fontSize: { base: '1.5rem', md: '1.75rem' },
```

Comment:
it does not have a fontWeight defined: after checking that a Figma bug shows in the inspectors 400 it really should be 700.